docs/InstallPrerequisites.md

```diff
@@ -22,16 +20,19 @@ Recommended way to install required tools and libs is to use [Homebrew](https://
 brew install node watchman cmake
 ```
 
-Download and run [Qt 5.9.1](https://download.qt.io/archive/qt/5.9/5.9.1/qt-opensource-mac-x64-5.9.1.dmg) installer for Mac OS. During installation process on `Select Components` screen check `Qt 5.9.1 / macOS` and `Qt 5.9.1 / Qt WebEngine` components. Finish installation process.
-
-Make sure that installed Qt's `qmake` tool is available in your PATH.
+Download and run [Qt 5.11.2](https://download.qt.io/archive/qt/5.11/5.11.2/qt-opensource-mac-x64-5.11.2.dmg) installer for Mac OS. During installation process on `Select Components` screen check `Qt 5.11.2 / macOS` and `Qt 5.11.2 / Qt WebEngine` components. Finish installation process.
 
 ### Ubuntu
 
 ```
 sudo apt-get install -y git cmake nodejs-legacy npm
 ```
 
-Download and run [Qt 5.9.1](https://download.qt.io/archive/qt/5.9/5.9.1/qt-opensource-linux-x64-5.9.1.run) installer for Linux. Finish installation process.
+Download and run [Qt 5.11.2](https://download.qt.io/archive/qt/5.11/5.11.2/qt-opensource-linux-x64-5.11.2.run) installer for Linux. Finish installation process.
+
+### All platforms
+
+Make sure that installed Qt's `qmake` tool is available in your PATH.
+On mac and linux you can add following to `.bash_profile`:
 
-Make sure that installed Qt's `qmake` tool is available in your PATH.
\ No newline at end of file
+`export PATH=$PATH:/Users/MyUser/Qt/5.11.2/clang_64/bin`
```

Comment:
perhaps it would be intuitive for anyone who made it this far, but since this section is `All platforms`, it might be helpful to add for linux users:
```
OR `export PATH=$PATH:/Users/MyUser/Qt/5.11.2/gcc_64/bin`
```

Reply:
I will put 
`export PATH=$PATH:/Users/MyUser/Qt/5.11.2/COMPILER_NAME/bin` :)

Reply:
bleh, haha. ok. sorry I hope my comment didn't come across as advocating busy work.